gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,340 @@
+
+Attempting to run Flex Gateway with an expired certificate in Connected Mode results in the following:
+
+* New replicas fail to download API configurations from Anypoint Platform, thereby becoming unusable. Existing clusters fail to reload.
+* Logs and metrics fail to upload to Anypoint Platform. Troubleshooting using Anypoint Platform is not possible.
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+Attempting to run Flex Gateway with an expired certificate in Local Mode results in the following:
+
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+If a certificate expires, you can still invoke `flexctl registration renew`, which updates the expired certificate.
+
+To avoid disruptions:
+
+. <<before-you-begin,Download the latest version of Flex Gateway.>>
+. <<renewal-cli-overview,Renew your Flex Gateway registration.>>
+. <<inspection-cli-overview,Verify (inspect) the success of the renewal process.>>
+
+[[before-you-begin]]
+== Before You Begin
+
+* xref:flex-install.adoc[Download Flex Gateway].
++
+To invoke the registration renewal `flexctl` command, download the latest version of Flex Gateway. It is not necessary to register and run this version of the gateway.
+
+[[renewal-cli-overview]]
+== Registration Renewal
+
+Flex Gateway includes the following CLI command to renew registration:
+
+----
+flexctl registration renew
+----
+
+The command renews an existing Flex Gateway registration with Runtime Manager. For authentication, you specify *either* user credentials or connected appl credentials. Authentication with user credentials requires the `--username` and `--password` flags. Authentication with connected app credentials requires the `--client-id` and `--client-secret` flags.
+
+Invoke the following `help` command for information about usage:
+
+[source,ssh]
+----
+flexctl registration renew --help
+----
+
+For Flex Gateway running in a container, invoke `help` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl mulesoft/flex-gateway registration renew --help
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl docker.io/mulesoft/flex-gateway registration renew --help
+----
+====
+
+Refer to the following topics for more information about renewal command usage:
+
+* <<renew-with-user-password,Renew Registration with User Credentials>>
+* <<renew-with-conn-app,Renew Registration with Connected App Credentials>>
+
+[[renew-with-user-password]]
+=== Renew Registration with User Credentials
```

Comment:
Rephrase as a task?

Reply:
This is one of two methods they can use to renew registration, so it in itself shouldn't be a task step. However, I made subtasks for each of these methods.

Reply:
I disagree. 

I think you can simplify and flatten this structure significantly.

In the intro, introduce the two methods and have the bullet list with links to each (as you do later).
Talk about when you would use one vs the other.

== Before You Begin
installation stuff

== Renew Registration with User Credentials
. Step 1
. Step 2

== Renew Registration with Connected App Credentials
. Step 1
. Step 2

== Get Help on Renew CLI Commands

Reply:
Fixed! I flattened the structure as suggested.